src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/BaseFactory.cs

```diff
@@ -0,0 +1,45 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using static Interop.CBApplication;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    internal class BaseFactory : ComponentFactoryBase
```

Comment:
Naming is too common, 
What is purpose of this class?
And Why do you need `base` naming?